src/model/page.urls.ts

```diff
@@ -10,14 +10,18 @@ export const PROMISE_TO_FILE: string = SEPARATOR + "promise-to-file";
 export const COMPANY_NUMBER: string = SEPARATOR + template.COMPANY_NUMBER;
 export const CHECK_COMPANY: string = SEPARATOR + template.CHECK_COMPANY;
 export const COMPANY_AUTH_PROTECTED_ROUTE: string = SEPARATOR + "company/:companyNumber/";
+export const CONFIRMATION_APPEND: string = "confirmation";
 
 /**
  * Company Auth protected routes
  */
 export const STILL_REQUIRED: string = COMPANY_AUTH_PROTECTED_ROUTE + template.STILL_REQUIRED;
-export const CONFIRMATION: string = COMPANY_AUTH_PROTECTED_ROUTE + "confirmation";
+export const CONFIRMATION: string = COMPANY_AUTH_PROTECTED_ROUTE + CONFIRMATION_APPEND;
+
 /**
  * URLs for redirects will need to start with the application name
  */
 export const PROMISE_TO_FILE_COMPANY_NUMBER: string = PROMISE_TO_FILE + COMPANY_NUMBER;
 export const PROMISE_TO_FILE_CHECK_COMPANY: string = PROMISE_TO_FILE + CHECK_COMPANY;
+export const PROMISE_TO_FILE_CONFIRMATION: string =
```

Comment:
Could this not just be PROMISE_TO_FILE_CONFIRMATION: string = PROMISE_TO_FILE + CONFIRMATION?
As above I believe the CONFIRMATON_APPEND constant is not required right now as the string "confirmation" only appears once